launchable/test_runners/raw.py

```diff
@@ -133,21 +160,27 @@ def parse_json(test_result_file: str) -> Generator[CaseEventType, None, None]:
             doc = json.load(f)
         default_created_at = datetime.datetime.now(datetime.timezone.utc).isoformat()
         for case in doc['testCases']:
-            test_path = parse_test_path(case['testPath'])
+            test_path_components: TestPath = case.get('testPathComponents', None)
```

Comment:
Don't we need to check `oneOf`? 
It seems `testPathComponents` will take over `testPath` when both are set.

Reply:
[65fa86c](https://github.com/launchableinc/cli/pull/764/commits/65fa86c19329c375cbd5c7649b601e334ba1dea3)